frontend/src/components/ChatBox/ChatBox.tsx

```diff
@@ -232,11 +236,29 @@ function ChatBox({
 					</span>
 				</div>
 				<div className="control-buttons">
-					<ExportPDFLink
-						messages={messages}
-						emails={emails}
-						currentLevel={currentLevel}
-					/>
+					<Suspense
+						fallback={
+							<ThemedButton
+								className={'chat-button chat-button-disabled'}
+								// eslint-disable-next-line @typescript-eslint/no-empty-function
+								onClick={() => {}}
+								ariaDisabled={true}
+								tooltip={{
+									id: 'export-chat-tooltip',
+									text: 'This button is still loading. Please wait...',
+								}}
+								tooltipAnchor="bottom-center"
```

Comment:
I think if we renamed this prop "tooltipPosition", it would help clarify the API.

Also then you'd need to switch "bottom" and "top".